### PR TITLE
Create a README file for the Compass Runtime Agent

### PR DESCRIPTION
**Pull request by majakurcius**
**Description**

Changes proposed in this pull request:

- Create a README.md file for the Compass Runtime Agent

#### **Related issue(s)**
--

### CODE REVIEW

File: components/compass-runtime-agent/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Prerequisites
+
+The Compass Runtime Agent requires Go 1.8 or higher.
+
+## Usage
+
+This section explains how to use the Compass Runtime Agent.
+
+### Start the Compass Runtime Agent
```
Comment (franpog859) [inline components/compass-runtime-agent/README.md]: I suggest to add a blank line below the title. Some markdown linters could get angry. Same thing after the 36th line, just before the bullet point

File: components/compass-runtime-agent/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+The Compass Runtime Agent has the following parameters:
+- **controllerSyncPeriod** is the time period between resyncing existing resources. Provide it in seconds. The default value is `60`.
+- **minimalConfigSyncTime** is the minimal time between synchronizing the configuration. Provide it in seconds. The default value is `300`.
+- **integrationNamespace** is the namespaces in which the resources are created. The default namespace is `kyma-integration`.
+- **gatewayPort** is the Application Gateway port. The default port is `8080`.
+- **insecureConfigurationFetch** specifies whether to fetch the configuration with disabled TLS verification. The default value is `false`.
+- **uploadServiceUrl** is the URL of the upload service. By default, it is an empty string.
+
+The Compass Runtime Agent also uses the following environment variables:
+- **DIRECTOR_URL**
```
Comment (franpog859) [inline components/compass-runtime-agent/README.md]: I think `RUNTIME_ID` and `TENANT` environment variables also should be there

File: components/compass-runtime-agent/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Parameters and environment variables
+
+The Compass Runtime Agent has the following parameters:
+- **controllerSyncPeriod** is the time period between resyncing existing resources. Provide it in seconds. The default value is `60`.
+- **minimalConfigSyncTime** is the minimal time between synchronizing the configuration. Provide it in seconds. The default value is `300`.
+- **integrationNamespace** is the Namespace in which the resources are created. The default Namespace is `kyma-integration`.
+- **gatewayPort** is the Application Gateway port. The default port is `8080`.
+- **insecureConfigurationFetch** specifies whether to fetch the configuration with disabled TLS verification. The default value is `false`.
+- **uploadServiceUrl** is the URL of the upload service. By default, it is an empty string.
```
Comment (tomekpapiernik) [inline components/compass-runtime-agent/README.md]:
```suggestion
- **uploadServiceUrl** is the URL of the Upload Service. By default, it is an empty string.
```

I think you're referring to the service by its proper name.

File: components/compass-runtime-agent/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+The Compass Runtime Agent also uses the following environment variables:
```
Comment (tomekpapiernik) [inline components/compass-runtime-agent/README.md]:
```suggestion
The Compass Runtime Agent uses these environment variables:
```

File: components/compass-runtime-agent/README.md (hunk 3 above)
Comment (majakurcius) [reply components/compass-runtime-agent/README.md]: I checked with my team. They say not to capitalize it.

**Pull request merged**